utils/auth.py

```
@@ -0,0 +1,63 @@
+from fastapi import Depends, HTTPException, status
+from fastapi.security import OAuth2PasswordBearer
+from jose import JWTError, jwt
+from dotenv import load_dotenv
+from utils.data import load_data
+import bcrypt
+
+load_dotenv()
+
+SECRET_KEY = os.getenv("SECRET_KEY", "fallback_secret")
+if not SECRET_KEY:
+    raise ValueError("SECRET_KEY 환경 변수가 설정되지 않았습니다.")
```

Ruff (0.14.13)

Avoid specifying long messages outside the exception class

(TRY003)